### ISSUE TITLE
Cannot access the model file 

### ISSUE THREAD

**Issue by berilerken** [JayChou-z/SiamIST#1]
Hi, I am from Turkey. Because of that I cannot reach the model file's website (htts://pan.baidu.com) that you provide. I want to try your model so, could you please provide me the model file? Thanks in advance.

**Comment by JayChou-z** [comment]
Of course, we can communicate via email. My email address is [6223110057@jiangnan.edu.cn](mailto:6223110057@jiangnan.edu.cn).

**Comment by berilerken** [comment]
I just downloaded the model file via baidu downloader website, but thank you so much.

**Comment by kiyaaabdian** [comment]
Cannot access the model file,can you upload it in google drive? also you don't have a main.py i don't know how to run it
